docs/unit-test-case.md

```
@@ -1,21 +1,272 @@
+| 算術期待値 (= `calcExpectedScore.finalScore`) | **344,497** | +51,377 |
+| MC 平均 (`runSimulation.mean`, SEED=42, iter=100) | **344,240** | +51,120 |
+| MC 最小 (`mcMin`) | 317,808 | +24,688 |
+| MC 最大 (`mcMax`) | 390,305 | +97,185 |
+| MC 標準偏差 (`stddev`) | 13,840 | — |
+
+### 縮小カバー率の計算ロジック (`calcShrinkCoverage`)
+
+1. **発動回数の上限**: `floor(notesCount / skill.count) = floor(407(notes_20を除外) / 20(Lv5の場合)) = 20 回`
+2. **スコアアップ対象の時間**: `20 回　※ 4秒 = 80秒`
+3. **最大縮小カバー率**: `80秒 / MonstaerGenetation 104秒`
+3. **縮小カバー率期待値**: `80秒*40% / MonstaerGenetation 104秒`
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 4; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)